src/controllers/findAllManagers.js

```diff
@@ -1,14 +1,12 @@
-
-import getManagers from '../helper/getManager'
+import getManagers from '../helper/getManager';
 const { Op } = require("sequelize");
 
 const listOfManager = async function (req, res,next) {
   const getAllManagers = await getManagers(req,res)
   res.status(200).send({
       title: 'Managers',
-      getAllManagers,
-    
-  })
-}
-export default listOfManager 
+      getAllManagers
```

Comment:
Why not not giving the names that are meaningful so that the returned data should be easy to access.

Reply:
The names and other data are returned from the function `getManagers(req, res)` found at [here](https://github.com/atlp-rwanda/c4-barefoot-backend/pull/37/files/ddbb180191caee71acc857717015b4dac9f992f8#diff-c1006b8b20059176f2b25f8e256b7a0b617297caee0bf754aadd1f2e80ad8890)
I hope I answered you.